src/components/BaseComponent.tsx

```diff
@@ -118,7 +127,17 @@ export function InfoboxContainer(props: InfoContainerProps): JSX.Element {
         padding: '1vw',
       }}
     >
-      <InfoBox siteId="4b14a8" fileHandler={fileHandler} />
+      <List
+        style={{
+          display: 'inline-flex',
+          flexDirection: 'row',
+          flexWrap: 'wrap',
+        }}
+      >
+        {siteIds.map((id) => (
+          <InfoBox siteId={id} fileHandler={fileHandler} />
+        ))}
```

Comment:
The list elements should have an unique key, otherwise we will get a React warning.

You can read about it here:
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key


```suggestion
        {siteIds.map((id) => (
          <ListItem key={id}>
            <InfoBox siteId={id} fileHandler={fileHandler} />
          </ListItem>
        ))}
```